src/Storefront/Framework/Routing/StorefrontResponse.php

```diff
@@ -22,7 +22,7 @@ public function getData(): array
         return $this->data;
     }
 
-    public function setData(?array $data): void
+    public function setData(array $data): void
```

Comment:
Hi @stephanniewerth 

Changing $data from array|null to a non-null array is a potential BC. Would you please only check the value in the method body once it's null at this moment.

Furthermore we are able to mark it deprecated that we only accept a non-null array in the next major release. For example:

```php
/**
 * @deprecated tag:v6.5.0 - parameter `$data` will be typed to `array` strictly 
 */
public function setData(?array $data): void

Reply:
Hi @trandangtri,

I implemented your comments. This should currently have no effect, just fix the error in `getData()`.